packages/credentials/src/presentation-exchange.ts

```diff
@@ -32,7 +32,13 @@ export class PresentationExchange {
   }): string[] {
     this.resetPex();
     const selectResults: SelectResults = this.pex.selectFrom(presentationDefinition, vcJwts);
-    return Array.from(new Set(selectResults.verifiableCredential as string[] ?? []));
+
+    // If errros the credentials provided didn't satisfy the requirement defined presentationDefinition object
```

Comment:
Small typo but possible to provide a better explanation based on this detail: https://github.com/TBD54566975/web5-js/issues/447
```suggestion
    // If errros the credentials provided didn't satisfy the requirement defined presentationDefinition object
```